templates/base.html

```diff
@@ -27,7 +27,8 @@ <h4 class="mx-2 hover:underline"><a href="/archive">Archive</a></h4>
         <h4 class="text-slate-50">{{SITE_TITLE}}</h4>
         <h4 class="text-slate-50">Made with <a class="hover:underline" href="https://github.com/kjaymiller/render_engine">Render Engine</a></h4>
         <h4 class="text-slate-50"><a class="hover:underline" href="https://github.com/kjaymiller/Python-Community-News/main/LICENSE.md" MIT Licensed</a></h4>
+        <h4 class="text-slate-50"><a class="hover:underline" href="https://github.com/kjaymiller/Python-Community-News/blob/main/CODE_OF_CONDUCT.md" Code of Conduct</a></h4>        
```

Comment:
Can you also add this  (with the `class="mx-2 hover:underline"` instead) in the `<nav></nav>` around Line 18.